gym/envs/box2d/car_racing.py

```diff
@@ -383,14 +383,15 @@ def render(self, mode='human'):
             self.human_render = True
             win.clear()
             t = self.transform
-            gl.glViewport(0, 0, WINDOW_W, WINDOW_H)
+            gl.glViewport(0, 0, 2 * WINDOW_W, 2 * WINDOW_H)
```

Comment:
This appears to cause issues with the rendering of the environment, the camera is locked to follow the car on the far right of the screen rather than the center. What was the intended purpose of this exact change?

Reply:
If true, thanks for tracking down changes to a PR!

Reply:
this code seems to work somewhat differently on osx and in linux. In osx, without this change, the viewport only occupies lower left corner of the window, whereas in linux I was able to reproduce behaviour described by @Qinusty. Let's revert for now, and I'll investigate what the problem is

Reply:
seems to be related to this: https://github.com/vispy/vispy/issues/99

Reply:
https://github.com/openai/gym/pull/1372/files